src/main/java/io/cdap/plugin/gcp/bigquery/sqlengine/util/BigQuerySQLEngineUtils.java

```diff
@@ -98,10 +108,46 @@ public static Long getNumRows(BigQuery bigQuery, String project, String dataset,
     return numRows;
   }
 
+  /**
+   * Creates an empty table with an empty schema to store records.
+   * <p>
+   * If the Engine Configuration specifies a TTL for tables, the table is created with the specified TTL.
+   *
+   * @param config   BigQuery SQL Engine Config instance
+   * @param bigQuery BigQuery client
+   * @param project  Project Name
+   * @param dataset  Dataset Name
+   * @param table    Table Name
+   */
+  public static void createEmptyTable(BigQuerySQLEngineConfig config,
+                                      BigQuery bigQuery,
+                                      String project,
+                                      String dataset,
+                                      String table) {
+
+    LOG.debug("Creating empty table {} in dataset {} and project {}", table, dataset, project);
+
+    // Define table name and create builder.
+    TableId tableId = TableId.of(project, dataset, table);
+    TableDefinition tableDefinition = StandardTableDefinition.of(com.google.cloud.bigquery.Schema.of());
+    TableInfo.Builder tableInfoBuilder = TableInfo.newBuilder(tableId, tableDefinition);
+
+    // Set TTL for table if needed.
+    if (!config.getRetainTables() && config.getTempTableTTLHours() > 0) {
+      long ttlMillis = TimeUnit.MILLISECONDS.convert(config.getTempTableTTLHours(), TimeUnit.HOURS);
+      long expirationTime = Instant.now().toEpochMilli() + ttlMillis;
+      tableInfoBuilder.setExpirationTime(expirationTime);
+    }
+
+    bigQuery.create(tableInfoBuilder.build());
+
+    LOG.debug("created empty table {} in dataset {} and project {}", table, dataset, project);
```

Comment:
nit: created -> Created to match the log statement from above.

Reply:
Oops, fixed.